html-forms/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form</h1>
+    <form action="">
+      Email
```

Comment:
text always needs to be in a `text element` don't leave it floating in the nether!